internal/cephfs/volume_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2020 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cephfs
+
+import (
+	"context"
+	"testing"
+)
+
+func Test_checkVersionIsGreater(t *testing.T) {
+	type args struct {
+		ctx     context.Context
+		version string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    bool
+		wantErr bool
+	}{
+		{
+			"valid lower ceph version which doesnot support resize",
+			args{
+				context.TODO(),
+				"14.2.7",
+			},
+			false,
+			false,
+		},
+		{
+			"invalid ceph version",
+			args{
+				context.TODO(),
+				"14.2",
+			},
+			false,
+			true,
+		},
+		{
+			"invalid ceph version",
+			args{
+				context.TODO(),
+				"abcd",
+			},
+			false,
+			true,
+		},
+		{
+			"invalid ceph version",
+			args{
+				context.TODO(),
+				"a.b.c",
+			},
+			false,
+			true,
+		},
+		{
+			"valid ceph version which supports resize",
+			args{
+				context.TODO(),
+				"14.2.8",
```

Comment:
Add a test for version 15.0 that does not have the resize support, and 15.2.2(?) that should get it

Reply:
am removing check for kernel version

Reply:
no need to check the kernel version? (ah, maybe Ceph version?)